hopp/simulation/technologies/layout/wind_layout.py

```diff
@@ -407,7 +496,17 @@ def reset_basic_grid(self,n_turbines):
                 self._set_system_layout()
                 return 
             else:
+                # just use `grid`` method
+                max_min_spacing_D = max(self.parameters.row_D_spacing,self.parameters.turbine_D_spacing)
+                max_min_spacing_m = max(interrow_spacing,intrarow_spacing)
```

Comment:
The "max_min" designation is throwing me off a little bit. Is there something we could use that's a bit more direct or at least that has less room for potential confusion when used later?

Reply:
Changed to `largest_min`